### PR TITLE
Add nemo action integration and complete integrations documentation

### LINKED ISSUE #107: pixelate.py: show help screen when called without options or filename

### LINKED ISSUE #108: Nemo script integration is not documented

### LINKED ISSUE #109: Add Nemo action integration

### LINKED ISSUE #112: Nemo is the cinnamon file manager, not GNOME’s

### PR DESCRIPTION
**Pull request by blackstream-x**
Closes #107,
closes #108,
closes #109,
closes #112

### CODE REVIEW

File: integration/pyxelate.nemo_action_template [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+Name=${name}
+Comment=${comment}
+Exec=${exec} %F
+Icon-Name=gtk-edit
```
Comment (Wikinaut) [inline integration/pyxelate.nemo_action_template:5]: What wll happen, if this icon file is missing?

**Review by Wikinaut — APPROVED** [review]
one comment. works for me.

File: pixelate.py [hunk 2]
```diff
@@ -287,12 +359,18 @@ def main(arguments):
     if selected_file:
         return start_matching_script(selected_file)
     #
-    logging.error("No (existing) file selected.")
+    toplevel = tkinter.Tk()
+    gui.InfoDialog(
```
Comment (Wikinaut) [inline pixelate.py:363]: I don't like the gui here.

**Review by Wikinaut — COMMENTED** [review]
added a further comment regarding the "gui-help".

File: integration/pyxelate.nemo_action_template (hunk 1 above)
Comment (blackstream-x) [reply integration/pyxelate.nemo_action_template:5]: Maybe that should also be configurable and default to a safe value

File: pixelate.py (hunk 2 above)
Comment (blackstream-x) [reply pixelate.py:363]: Will change that back. Please also have a look at https://github.com/blackstream-x/pyxelate/discussions/113 – I am thinking of separating the install part from the pixelate.py script

File: integration/pyxelate.nemo_action_template (hunk 1 above)
Comment (blackstream-x) [reply integration/pyxelate.nemo_action_template:5]: The question is: which icon can be assumed to be safe?
Maybe have a list of desired icon names, try to find them in /usr/share/icons and return the first one found?

**Pull request merged**